enhavo/app-bundle/0.9/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "bundles": {
+        "Enhavo\\Bundle\\AppBundle\\EnhavoAppBundle": ["all"]
+    },
+    "container": {
+        "locale": "en"
+    },
+    "env": {
+        "MAILER_FROM": "no-reply@email.com",
+        "MAILER_TO": "my@email.com",
+        "MAILER_NAME": "enhavo",
+        "MAILER_DELIVERY_ADDRESS": "my@email.com"
```

Comment:
These variables are never used. You should make sure to add those to `enhavo_app.yaml`